style-responsive.css

```diff
@@ -287,9 +287,26 @@
 		background-color: transparent;
   }
 
+  .featured-deck {
+    margin: 1em 0;
+  }
+
+  .related-story-title-wrapper {
+    height: 230px;
```

Comment:
Setting fixed heights on .related-story-title-wrapper can cause overflow when story titles are too long.  You might need to modify the bootstrap .row and .span classes within that section to achieve equal height rows; alternatively consider utilizing matchheight.js for this.

![screen shot 2018-01-16 at 10 43 39 am](https://user-images.githubusercontent.com/1649216/35000659-96666a3c-faaa-11e7-8c5f-960464e23bb0.png)

Reply:
I'll fix up the fixed height and add match height in the next pull request, along with a fallback image for stories.